L1Trigger/L1THGCal/interface/veryfrontend/HGCalVFECompressionImpl.h

```diff
@@ -2,14 +2,28 @@
 #define __L1Trigger_L1THGCal_HGCalVFECompressionImpl_h__
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
 
 class HGCalVFECompressionImpl
 {
 
  public:
   HGCalVFECompressionImpl(const edm::ParameterSet& conf);
+
+  uint32_t bitLength(uint32_t x);
```

Comment:
This should rather be a `private` method

Reply:
Done!